.github/workflows/docker-build-publish.yml

```diff
@@ -20,15 +20,17 @@ jobs:
     permissions:
       contents: write
       packages: write
-    uses: celestiaorg/.github/.github/workflows/reusable_dockerfile_pipeline.yml@v0.2.8 # yamllint disable-line rule:line-length
+    uses: celestiaorg/.github/.github/workflows/reusable_dockerfile_pipeline.yml@v0.3.1
     with:
       dockerfile: Dockerfile
+    secrets: inherit
 
   docker-txsim-build:
     permissions:
       contents: write
       packages: write
-    uses: celestiaorg/.github/.github/workflows/reusable_dockerfile_pipeline.yml@v0.2.8
+    uses: celestiaorg/.github/.github/workflows/reusable_dockerfile_pipeline.yml@v0.3.1
     with:
       dockerfile: docker/Dockerfile_txsim
       packageName: txsim
+    secrets: inherit
```

Comment:
[Question] Mind illuminating why this secrete inheritance is needed? Is this required by the new version of `reusable_dockerfile_pipeline`?

Reply:
Apparently yes per this [Slack thread](https://celestia-team.slack.com/archives/C02PEA73QM6/p1706543641089669?thread_ts=1706540846.259989&cid=C02PEA73QM6).